generators/asset-modifier.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+const fs = require('fs');
+const colors = require('colors');
+
+module.exports.addComponentToModule = function(filename, variableSuffix, importPrefix, importSuffix, componentName, camelName, capitalName) {
+  // making an attempt to fit in with yeoman
+  console.log('   modify'.green, filename);
+
+  // read the file
+  let content = fs.readFileSync(filename, 'utf8');
+
+  // TODO use this for safety https://github.com/sindresorhus/detect-newline
+  const eol = '\r\n';
```

Comment:
had to change to `\n` to make it work... any risks?

Reply:
Not a huge risk, git does a decent job of mitigating windows/linux line breaks.  The TODO comment is there as a reminder to completely remove the risk by detecting the line break style and using it.

I develop on Windows so it at worst, it might create a line that ends with "\r\n\n", which I think would be removed when I commit anyway...